src/components/rating/subcomponents/RateCounter.jsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import RateCounterItem from './RateCounterItem';
 
-const RateCounter = ({counter}) => {
-  if (counter === undefined) {
-    return null;
-  }
-
-  return <div className="sg-rate-box__counter">{counter}</div>;
-};
+const RateCounter = ({activeText, counterText, showActiveText}) =>
+  <div className="sg-rate-box__counter">
+    <RateCounterItem text={counterText} hidden={showActiveText} />
+    <RateCounterItem text={activeText} hidden={!showActiveText} />
```

Comment:
Hm.. I don't understand why we need this component. Why do we need two elements? Why just not:

```
<div className="sg-rate-box__counter">
   <RateCounterItem text={text} />
</div>
```

Reply:
if you have text and replace it will bounce

Reply:
ah, right we talked about it 👍